### ISSUE TITLE
[bitnami/kube-prometheus] Incorrect label used by service selector

### ISSUE THREAD

**Issue by kenanwarren** [bitnami/charts#8095]
**Which chart**:
kube-prometheus 6.3.2

**Describe the bug**
As part of the [0.52.0 release](https://github.com/prometheus-operator/prometheus-operator/blob/main/CHANGELOG.md) of the prometheus operator it no longer applies the app label to the statefulsets/pods. This label is used by a [couple](https://github.com/bitnami/charts/blob/master/bitnami/kube-prometheus/templates/prometheus/thanos-service.yaml#L37) [different](https://github.com/bitnami/charts/blob/master/bitnami/kube-prometheus/templates/prometheus/service.yaml#L42) service YAMLs (maybe others). I believe this is causing the pods to not get targeted by the service.

**To Reproduce**

1. Install the v6.3.2 chart
2. Run `kubectl describe svc kube-prometheus-prometheus-thanos -n installed-namespace`
3. Verify `endpoints` doesn't have a value.

**Expected behavior**
That when I run `kubectl describe svc kube-prometheus-prometheus-thanos -n installed-namespace` there is at least one valid value for `endpoints`.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.7.0", GitCommit:"eeac83883cb4014fe60267ec6373570374ce770b", GitTreeState:"clean", GoVersion:"go1.17"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.5", GitCommit:"aea7bbadd2fc0cd689de94a54e5b7b758869d691", GitTreeState:"clean", BuildDate:"2021-09-15T21:10:45Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"21+", GitVersion:"v1.21.2-eks-0389ca3", GitCommit:"8a4e27b9d88142bbdd21b997b532eb6d493df6d2", GitTreeState:"clean", BuildDate:"2021-07-31T01:34:46Z", GoVersion:"go1.16.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
Going by the equivalent `prometheus-community` chart I think we should migrate the selector from `app` to `app.kubernetes.io/name`

**Comment by javsalgar** [comment]
Hi,

Thank you so much for reporting! As you were the one that discovered the issue, would you like to contribute with a PR? If not, we will work on it.

**Comment by kenanwarren** [comment]
Sure I'll take a look at it in a couple of days.

**Comment by js-timbirkett** [comment]
Created a PR for this: #8104 - didn't see this Issue :)

**Comment by viceice** [comment]
fixed in #8104, so can be closed
